Gemfile

```diff
@@ -9,3 +9,10 @@ unless RUBY_PLATFORM
   gem 'pry-stack_explorer'
   gem 'pry-rescue'
 end
+
+# Gem a dependency of Capybara, new version of rack 2.0 onwards require ruby 2.2 and above.
+if RUBY_VERSION < "2.2.2"
```

Comment:
So yeah, that second one is incorrect...so if 2 gets to 2.10.x we'll have problems.